hugo/layouts/partials/card.html

```diff
@@ -0,0 +1,11 @@
+<article class="cards__element cards__element_category">
```

Comment:
https://en.bem.info/methodology/naming-convention/#two-dashes-style another naming convention, apparently. I thought the `--` was standard. Since we have no modifiers so far feel free to chose between one or the other.